apps/drec-ui/src/pages/LoginPage/LoginPage.effects.ts

```diff
@@ -0,0 +1,70 @@
+import { useUserLogin } from '../../api';
+import { useNavigate } from 'react-router';
+import { UserModalsActionsEnum, useUserModalsDispatch } from '../../context';
+import { UnpackNestedValue } from 'react-hook-form';
+import { GenericFormProps } from '../../core';
+
+import * as Yup from 'yup';
+
+export type TUserLoginFormValues = {
+    username: string;
+    password: string;
+};
+
+export const INITIAL_FORM_VALUES: TUserLoginFormValues = {
+    username: '',
+    password: ''
+};
+
+export const useLogInPageEffects = () => {
+    const navigate = useNavigate();
+    const dispatchModals = useUserModalsDispatch();
+    const useUserLogInFormConfig = (
+        formSubmitHandler: (values: UnpackNestedValue<TUserLoginFormValues>) => void
+    ): GenericFormProps<TUserLoginFormValues> => {
+        return {
+            buttonFullWidth: true,
+            buttonText: 'Login',
```

Comment:
Do we consider using localization from the beginning ?

Reply:
No localization required